utils/src/main/kotlin/ArchiveUtils.kt

```diff
@@ -112,6 +118,35 @@ fun File.unpack7Zip(targetDirectory: File) {
  */
 fun File.unpackZip(targetDirectory: File) = ZipFile(this).unpack(targetDirectory)
 
+/**
+ * A list with file names that are expected to be contained in a Debian package archive file.
+ */
+private val DEBIAN_ARCHIVE_CONTENT = listOf("data.tar.xz", "control.tar.xz")
+
+/**
+ * Unpack the [File] assuming it is a Debian archive. A Debian archive is an ar archive that contains itself two tar
```

Comment:
To me "itself" sounds odd, consider either dropping it or saying "..which in turn contains two tar.."
(Could be just my english)

Also maybe: "...files with metadata and the actual content of the package." -> 
"...files, meta data and the actual content of the package.